### PR TITLE
Doc: update to v11

### PR DESCRIPTION
**Pull request by mart-e**
- Use python 3.5
- Refer to correct page of the python documentation
- Change URLs
- Remove old bazar to git (was intended for the 8.0)
- Remove outdated setup_dev script (it was intended for odoo developers but if you are not able to make a git clone, you are going to have a bad time later)

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -178,7 +178,7 @@
 todo_include_todos = False
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/2/', None),
+    'python': ('https://docs.python.org/3/', None),
```
Comment (mart-e) [inline doc/conf.py]: not sure where this is used

Comment (xmo-odoo) [reply doc/conf.py]: Ref links to Python objects and docs, not sure if there are still extant ones.

File: doc/setup/install.rst [hunk 2]
```diff
@@ -207,6 +207,11 @@ RPM
 
 .. warning::
 
+    As of 2017, Fedora 26 is recommanded. CentOS does not have the minimum
```
Comment (Yenthe666) [inline doc/setup/install.rst]: @mart-e this should be `recommended` instead of `recommanded`

File: doc/setup/install.rst [hunk 3]
```diff
@@ -207,6 +207,11 @@ RPM
 
 .. warning::
 
+    As of 2017, Fedora 26 is recommended. CentOS does not have the minimum
+    python requirements (3.5) for Odoo 11.0.
```
Comment (rim-odoo) [inline doc/setup/install.rst]: [typonazi] **P**ython [/typonazi]

File: doc/setup/install.rst [hunk 4]
```diff
@@ -351,18 +356,23 @@ Installing dependencies
...
 
-* Python 2.7.
+* Python 3.5+.
+
+  - on Linux and OS X, using your package manager if not installed by default
+
+    .. note:: on some system, ``python`` commands refers to Python 2 (outdated)
+              or to Python 3 (supported). Make sure you are using the right
+              version and that the alias ``python3`` is present in your
+              :envvar:`PATH`
```
Comment (rim-odoo) [inline doc/setup/install.rst]: idem for pip? pip may refer to pip2 or pip3

File: doc/setup/install.rst [hunk 5]
```diff
@@ -351,18 +356,23 @@ Installing dependencies
 
 Source installation requires manually installing dependencies:
 
-* Python 2.7.
+* Python 3.5+.
+
+  - on Linux and OS X, using your package manager if not installed by default
+
+    .. note:: on some system, ``python`` commands refers to Python 2 (outdated)
```
Comment (rim-odoo) [inline doc/setup/install.rst]: Depending on your distribution,  ``python`` command (without a s) refers to Python 2...

Comment (xmo-odoo) [reply doc/setup/install.rst]: Isn't that what the note says?

**Pull request merged**